docs/libs/gelly_guide.md

```diff
@@ -693,6 +693,9 @@ Currently, the following parameters can be specified:
 * <strong>Number of Vertices</strong>: Accessing the total number of vertices within the iteration. This property can be set using the `setOptNumVertices()` method.
 The number of vertices can then be accessed in the gather, sum and/or apply functions by using the `getNumberOfVertices()` method. If the option is not set in the configuration, this method will return -1.
 
+* <strong>Neighbor Direction</strong>: By Default values are gathered from the out neighbors of the Vertex. This can be modified
```

Comment:
default without capital D, set direction should be written without capital s and in between inverted commas (i.e. `setDirection()`)

It would be nice to have an example in the docs, showing users how to play with this option; check the number of vertices example for inspiration :)